tests/automation_framework/src/libs/test_base.py

```diff
@@ -14,14 +14,12 @@ def __init__(self):
         self.uri_client = env.uri_client
         self.build_request_output = {}
 
-    def setup_and_build_request_register(self, input_file):
-        pre_test_output = pre_test_worker_env(input_file)
+    def setup_and_build_request_worker_register(self, input_file):
```

Comment:
Renamed the setup_and_build_request_register to setup_and_build_request_worker_register